assets/tests/hooks/useVehicleForNotification.test.tsx

```diff
@@ -20,85 +24,11 @@ jest.mock("../../src/helpers/googleTagManager", () => ({
   tagManagerEvent: jest.fn(),
 }))
 
-const ghostData: GhostData = ghostDataFactory.build({
-  id: "ghost-5678",
-  direction_id: 0,
-  route_id: "39",
-  trip_id: "12345678",
-  headsign: "Forest Hills",
-  block_id: "S12-34",
-  run_id: "123-4567",
-  via_variant: "3",
-  layover_departure_time: null,
-  scheduled_timepoint_status: {
-    timepoint_id: "shunt",
-    fraction_until_timepoint: 0.3342478185618462,
-  },
-  scheduled_logon: null,
-  route_status: "on_route",
-  block_waivers: [],
-  current_piece_first_route: null,
-  current_piece_start_place: null,
-})
+const runId = "run_id"
```

Comment:
suggestion(non-blocking): I added a `runIdFactory` that might help standardize runId formats across tests?